.github/workflows/check-packages.yml

```diff
@@ -0,0 +1,56 @@
+name: Check packages
+on:
+  pull_request:
+
+jobs:
+  dependency-mismatches:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '14'
+
+      - uses: actions/github-script@v6
+        with:
+          script: |
+            const syncpackVersion = require('./package.json').devDependencies.syncpack;
+
+            if (typeof syncpackVersion !== 'string') {
+              core.setFailed("Please check 'package.json', it should include 'syncpack' in 'devDependencies'")
+              return
+            }
+
+            core.exportVariable('SYNCPACK_VERSION', syncpackVersion);
+
+      - run: |
+          npx syncpack@$SYNCPACK_VERSION list-mismatches
```

Comment:
nit: Is there sense to align second script to use the same approach? Or align this script to do the same.

Reply:
right, maybe :D , we cannot do this for beachball because we cannot set npx version in package.json alias

```
"checkchange": "beachball check  --scope \"!packages/fluentui/*\" --changehint \"Run 'yarn change' to generate a change file\"",

↓↓↓ this would use whatever is latest

"checkchange": "npx beachball check  --scope \"!packages/fluentui/*\" --changehint \"Run 'yarn change' to generate a change file\"",
```

What can be done though is following:

- update config
![image](https://user-images.githubusercontent.com/1223799/157634125-0b8f9283-5619-4479-8194-37e1efc18f90.png)
- remove redundant aliases

```diff
- "checkchange": "beachball check  --scope \"!packages/fluentui/*\" --changehint \"Run 'yarn change' to generate a change file\"",
```

- use same npx approach within action
```
cp scripts/beachball/.beachballrc.base.json .beachballrc.json
npx beachball@$BEACHBALL_VERSION  check
```

lemme know what you think and I'll do it in another PR

Reply:
Let's do it in a separate PR, but it was more curiosity than a real need 🐱

Reply:
that curiosity will lead to simplification of our aliases so all good ;)